sources/Sifrei_Devarim/Sifrei_Devarim_reparse.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+# for a script located two directories below this file
+p = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.insert(0, p)
+from sources.local_settings import *
+sys.path.insert(0, SEFARIA_PROJECT_PATH)
+os.environ['DJANGO_SETTINGS_MODULE'] = "local_settings"
+from sources.functions import *
+import re
+import codecs
+
+def get_parsed_text():
+    with open('Sifrei_Devarim_reparse_text.txt') as my_file:
+        lines = my_file.readlines()
+    piska_box = []
+    final_text = []
+    oops = []
```

Comment:
Not the best variable name. I see you're tracking some sort of error with this, but it isn't immediately clear what error that is.